src/modules/dashboard/components/Dialogs/StartStreamDialog/StartStreamDialogForm.tsx

```diff
@@ -0,0 +1,179 @@
+import { FormikProps } from 'formik';
+import React, { Fragment, useCallback, useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import { COLONY_TOTAL_BALANCE_DOMAIN_ID } from '~constants';
+import Button from '~core/Button';
+import { DialogSection } from '~core/Dialog';
+import { Annotations, FormSection, Toggle } from '~core/Fields';
+import Heading from '~core/Heading';
+import MotionDomainSelect from '~dashboard/MotionDomainSelect';
+import { Colony } from '~data/index';
+import { ValuesType } from '~pages/ExpenditurePage/types';
+
+import FundingSource from './FundingSource';
+import { FormValues } from './StartStreamDialog';
+import StreamingDetails from './StreamingDetails';
+import {
+  user,
+  endDate,
+  endDateTime,
+  startDate,
+} from './StreamingDetails/constants';
+import styles from './StartStreamDialog.css';
+
+const displayName = 'dashboard.StartStreamDialog.StartStreamDialogForm';
+
+const MSG = defineMessages({
+  title: {
+    id: 'dashboard.StartStreamDialog.StartStreamDialogForm.title',
+    defaultMessage: 'Create a Motion to start Stream',
+  },
+  description: {
+    id: 'dashboard.StartStreamDialog.StartStreamDialogForm.title',
+    defaultMessage: `To create this Streaming Payment you need to have Adminstration permissions or get collective approval first. To get collective approval, you can create a Motion.`,
+  },
+  createMotion: {
+    id: 'dashboard.StartStreamDialog.StartStreamDialogForm.title',
+    defaultMessage: 'Create Motion',
+  },
+  streamingDetails: {
+    id: 'dashboard.StartStreamDialog.StartStreamDialogForm.streamingDetails',
+    defaultMessage: 'Streaming details',
+  },
+  fundingSource: {
+    id: 'dashboard.StartStreamDialog.StartStreamDialogForm.fundingSource',
+    defaultMessage: 'Funding Source {count}',
```

Comment:
I noticed the fundingSource count is displayed differently in Figma. Please update to match the designs:
<img width="592" alt="Screen Shot 2022-10-03 at 7 51 58 PM" src="https://user-images.githubusercontent.com/71563622/193645006-2b8fa226-2402-4df2-9bcf-05fb48e14246.png">
Also, please double check with Arren to confirm if a single streaming should display a number or not. Thanks!

Reply:
It would be great to keep in consistent, with the format `#. {Name}` for all of them. e.g. `1. Funding Source`.

Reply:
Added change.